control_panel_api/settings/base.py

```diff
@@ -124,5 +124,6 @@
 LOGS_BUCKET_NAME = os.environ.get('LOGS_BUCKET_NAME', 'moj-analytics-s3-logs')
 IAM_ARN_BASE = os.environ.get('IAM_ARN_BASE', '')
 K8S_WORKER_ROLE_NAME = os.environ.get('K8S_WORKER_ROLE_NAME', '')
-
+SAML_PROVIDER_ARN = os.environ.get('SAML_PROVIDER_ARN', '')
+K8S_WORKER_ROLE_ARN = os.environ.get('K8S_WORKER_ROLE_ARN', '')
```

Comment:
This is not required as we have both `IAM_ARN_BASE` and `K8S_WORKER_ROLE_NAME` here already.

[See existing `services.create_app_role()` implementation](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/api/control_panel_api/services.py#L56).